integration-tests/reactive-messaging-kafka/pom.xml

```diff
@@ -15,6 +15,8 @@
 
     <properties>
         <maven.compiler.parameters>true</maven.compiler.parameters>
+        <!-- needs to wait for smallrye-reactive-messaging-health to update MP health to 3.0 -->
+        <enforcer.skip>true</enforcer.skip>
```

Comment:
Remove this once verified that rest will pass tests.

Reply:
Removing this as there is also a failure in the native image build tests which depends on the SR reactive messaging update.